src/ui-schema.ts

```diff
@@ -155,7 +155,7 @@ export type AnyOfControlOptions = OneOfControlOptions
 export type TextControlType = "multiline" | "password" | "singleline"
 
 export type TextControlOptions = {
-  tooltip?: string
+  tooltip?: React.ReactNode
```

Comment:
why is this change necessary?

Reply:
so we can render HTML here as discussed in [the thread](https://greatexpectationslabs.slack.com/archives/C05FV397N86/p1718743452516369)

Reply:
formitem's tooltip is already typed as a ReactNode, though

Reply:
what is the issue with it?